src/main/webapp/mapScript.js

```diff
@@ -89,53 +96,66 @@ function addNewTableItem(name, index) {
   tabel.appendChild(newPlace);
 }
 
-function deletePlace(contentId) {
-  let tableItem = document.getElementById("place" + contentId);
+function getActualIndex(placeIndex) {
```

Comment:
in what cases can they (getActualIndex and placeIndex ) differ? if the user swaps the markers?

Reply:
Or if the user deletes one

Reply:
I thought that globalIndex may already resolve those cases? 🤔 It just a bit strange for me, because, for example
deletePlace created with "index" variable and at the same time new element added to the markersArray with the same "index"